src/OpenTelemetry/Trace/ActivityProcessor.cs

```diff
@@ -15,8 +15,10 @@
 // </copyright>
 
 using System;
+using System.Collections.Generic;
```

Comment:
@eddynaka Are these new usings actually needed? Diff isn't showing any code changing 🤷

Reply:
strange O_O. will remove! thanks for reviewing again!